app/utilities/utils.py

```diff
@@ -1,10 +1,9 @@
 from pydantic import ValidationError
-
+from flask import jsonify
```

Comment:
What is jsonify being used for?

Reply:
I removed the logic I was using that for I just forgot remove the import statement I'd push a commit to rectify that

Reply:
Sure sure